### PR TITLE
docs(3rd-party): Add section for third party tools

### PR DESCRIPTION
**Pull request by lucasssvaz**
## Description of Change

Add section for 3rd party tools in the documentation. Also fixes issues with dependencies in Python 3.13

## Tests scenarios

Locally

## Related links

#10045

### CODE REVIEW

File: docs/en/thirdparty/wokwi.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.. note::
+  This is a work in progress documentation and we will appreciate your help! We are looking for contributors!
+
+About
+-----
+
+Wokwi is an online Electronics simulator. You can use it to simulate Arduino, ESP32, and many other popular boards, parts and sensors.
+
+The advantages of using Wokwi inlcude:
```
Comment (P-R-O-C-H-Y) [inline docs/en/thirdparty/wokwi.rst:16]:
```suggestion
The advantages of using Wokwi include:
```

**Review by P-R-O-C-H-Y — APPROVED** [review]
Just a small typo, idk if pre-commit will fix automatically

**Comment by Jason2866** [comment]
@me-no-dev Probably accidentally closed by deleting the merged branch

**Comment by lucasssvaz** [comment]
@Jason2866 The new PR is here https://github.com/espressif/arduino-esp32/pull/10749

**Pull request closed without merging**